### PR TITLE
[Doc]Note on IAM role

### PR DESCRIPTION
**Pull request by ArvinthC3000**
### :chains: Related Resources
Jira: https://chefio.atlassian.net/browse/CHEF-5508

### :white_check_mark: Checklist

**All PRs** must tick these:

- [ ] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [ ] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [ ] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [ ] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [ ] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [ ] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ Follow the steps below to deploy Chef Automate High Availability (HA) on AWS (Am
...
 - We need three private and three public subnets in a vpc (1 subnet for each AZ). As of now, we support a dedicated subnet for each AZ.
 - We recommend creating a new vpc. And Bastion should be in the same VPC.
-- Get AWS credentials (`aws_access_key_id` and `aws_secret_access_key`) with privileges like: `AmazonS3FullAccess`, `AdministratorAccess`.
+- Attach IAM role to the Bastion with `AmazonS3FullAccess`, `AdministratorAccess` privileges or get AWS user credentials with the same privileges. Click [here](/automate/ha_iam_user/) to learn more about creating IAM Users.
 
-    Set these in `~/.aws/credentials` in Bastion Host:
+Set the AWS user credentials in `~/.aws/credentials` in Bastion Host:
 
     ```bash
     sudo su -
```
Comment (vivek-yadav) [inline components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:33]: why `sudo su -`  is needed ?

File: components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md [hunk 2]
```diff
@@ -245,7 +251,7 @@ After successful deployment, proceed with following...
     instance_count = "0"
 [aws]
   [aws.config]
-    profile = "default"
+    profile = "default"   #This should be commented incase if IAM role is attached
```
Comment (vivek-yadav) [inline components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md:254]:
```suggestion
    profile = "default"   # This should be commented incase if IAM role is attached
```

Comment (ArvinthC3000) [reply components/docs-chef-io/content/automate/ha_aws_managed_deploy_steps.md]: Done

File: components/docs-chef-io/content/automate/ha_aws_deploy_steps.md (hunk 1 above)
Comment (ArvinthC3000) [reply components/docs-chef-io/content/automate/ha_aws_deploy_steps.md:33]: As we are executing chef-automate with sudo permission, terraform will try to access credentials from /root folder. And only root user will have permission to write in that folder. So we need that

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**